### ISSUE TITLE
DDbDP texts with no TM number

### ISSUE THREAD

**Issue by hcayless** [papyri/idp.data#504]
Working on the migration of DDb texts, and the following have no TM ids:
```txt
sb.1.2025b.xml
sb.1.2025a.xml
sb.1.5276A.xml
sb.1.5276B.xml
sb.1.222.xml
sb.8.9861c.xml
sb.8.9861b.xml
sb.8.9861a.xml
sb.10.10651-I.xml
sb.14.11527o.xml
o.mich.4.1140.xml
o.mich.1.662.xml
o.mich.1.661.xml
o.mich.1.658.xml
o.mich.1.656.xml
o.mich.1.657.xml
p.tebt.2.267v.xml
p.tebt.2.277r.xml
p.tebt.3.2.880.xml
p.lond.2.188v.xml
o.bodl.2.2422.xml
o.petr.467.xml
p.alex.inv291.xml
upz.1.67b.xml
upz.1.pg603-5.xml
p.hamb.1.26.xml
p.harrauer.37.xml
t.mom.louvre.739.xml
p.leid.inst.1.13.xml
p.leid.inst.1.pg221.xml
cpr.3.84.xml
cpr.3.45.xml
cpr.3.50.xml
cpr.3.82.xml
cpr.3.41.xml
cpr.3.57.xml
cpr.3.43.xml
cpr.3.94.xml
cpr.3.31.xml
cpr.3.26.xml
cpr.3.27.xml
cpr.3.37.xml
cpr.3.9.xml
cpr.3.8.xml
cpr.3.39.xml
cpr.3.12.xml
cpr.3.98.xml
cpr.3.106.xml
cpr.3.99.xml
cpr.3.58.xml
cpr.3.70.xml
cpr.3.105.xml
cpr.3.104.xml
cpr.3.65.xml
cpr.3.59.xml
cpr.3.100.xml
cpr.3.74.xml
cpr.3.62.xml
cpr.3.63.xml
p.mich.4.2.359F.xml
zpe.160.224_3.xml
p.oxy.4.695v.xml
p.oxy.3.407v.xml
p.oxy.10.1244v.xml
p.oxy.2.209r.xml
p.oxy.2.229v.xml
p.oxy.47.3367-r-1.xml
chla.22.722.xml
chla.47.1436.xml
chla.5.288.xml
chla.43.1254.xml
chla.9.400.xml
p.ryl.2.273a.xml
p.petr.2.21.xml
p.petr.3.93.xml
p.petr.3.59.xml
p.michael.53.xml
p.koeln.12.484lin1_4.xml
p.wuerzb.1.2r.xml
p.batav.AppE.xml
p.bingen.appendix-pg166-7-1.xml
p.bingen.appendix-pg166-7-3.xml
p.bingen.appendix-pg166-7-2.xml
o.douch.4.nonInv.1993.xml
o.douch.4.Inv.88-133.xml
p.amh.2.18r.xml
sb.kopt.5.2313.xml
sb.kopt.5.2314.xml
apf.58.236.xml
p.mon.apollo.1.34.xml
p.oslo.3.178.xml
p.nekr.36.xml
psi.8.982-r.xml
psi.9.1087-v.xml
psi.9.1083-r.xml
psi.9.1086-v.xml
psi.9.1095-v.xml
psi.7.870H.xml
psi.7.846-v.xml
psi.7.870D.xml
psi.7.870G.xml
psi.2.119.xml
psi.2.142.xml
psi.14.1452.xml
psi.14.1451.xml
psi.14.1450.xml
p.hal.7.xml
p.fay.17.xml
pap.biling.6.xml
p.harr.2.211Fr.xml
p.ross.georg.5.12dadd.xml
p.ross.georg.5.4add.xml
p.ross.georg.5.25add.xml
p.ross.georg.4.pg8.xml
o.theb.89.xml
p.clackson.45.xml
p.clackson.34.xml
p.clackson.22.xml
c.ep.lat.AppVindol.xml
basp.50.68.xml
p.nyu.2.2.xml
p.nyu.2.32.xml
stud.pal.5.124r_1.xml
```

**Comment by hcayless** [comment]
Some of these might be stubs--the script I'm running doesn't account for stubs that have something inside their body `<ab/>` so some of those may have snuck through (though they probably ought to have TM numbers anyway).

**Comment by jcowey** [comment]
Yes they too (stubs) should have TM numbers. I will go through the list and make corrections.

**Comment by LaviniaFerretti** [comment]
I did a quick XQUERY search and found the following ones still without TM (mostly stubs):

chla.22.719
chla.4.265
chla.47.1459
chla.7.323
chla.7.324
chr.wilck.293
chr.wilck.90
cpr.1.20-pg110
o.douch.4.P.KellisInv.1383
p.amh.2.166
p.amst.1.32-4_10
p.amst.1.41-r-2
p.amst.1.41-r-3
p.amst.1.41-v
p.athen.51
p.batav.4
p.batav.7A
p.berl.brash.4
p.berl.thun.2
p.berl.thun.4
p.cair.masp.3.67316r
p.cair.preis.34
p.cair.preis.6
p.charite.9b
p.eleph.wagner.1.81
p.flor.1.71
p.flor.1.87
p.fuad.i.univ.App_I_3
p.grenf.2.48
p.ital.1.24
p.ital.1.3
p.ital.1.9
p.kell.1.19a
p.kell.1.19a_App
p.leit.5a
p.lond.1.112-pxviii
p.lond.1.138-pxix
p.lond.1.25
p.lond.1.31-p15
p.lond.2.288
p.lond.2.378
p.lond.2.386-pgxxxvi
p.lond.3.1017
p.lond.3.1101
p.lond.3.1124Apglviii
p.lond.3.11691+2pglxii
p.lond.3.1260pglxix
p.lond.3.1271pglxx
p.lond.3.1284A
p.lond.3.1311
p.lond.3.682
p.lond.3.889a
p.mich.mchl.12
p.oxy.1.190
p.oxy.2.326
p.oxy.4.802
p.paris.23r
p.paris.55
p.paris.63
p.petr.2.8-1
p.petr.3.118
p.petr.3.144
p.petr.3.146
p.petr.3.20-r-4+v
p.petr.3.24-FrA
p.petr.3.pg18
p.prag.2.123
p.prag.varcl.NS.10
p.prag.varcl.NS.17
p.prag.varcl.NS.3
p.princ.1.13
p.princ.3.134
p.rein.1.37
p.rein.1.39
p.ross.georg.2.1
p.ryl.2.278
p.tebt.1.86v
p.vind.bosw.14
psi.4.398
psi.9.1010A
psi.9.1010B
psi.9.1010C
psi.9.1010D
psi.9.1010E
psi.9.1064-r-I
psi.9.1064-r-II
psi.congr.xx.8
psi.congr.xxi.18
sb.12.10834
sb.12.10996
sb.12.11176
sb.3.6277
sb.3.6760
sb.3.6766
sb.3.6767
sb.3.6921
sb.3.7196
sb.4.7373
sb.4.7407
sb.4.7414
sb.4.7416
sb.4.7417
sb.5.7608
sb.5.8072
sb.6.9010
sb.6.9312
sb.6.9355
sb.6.9408-2
sb.6.9409-7
sb.6.9410-7
sb.6.9480-6dupl
sb.6.9620-7
sb.6.9620-8
stud.pal.14.8C
stud.pal.20.95
stud.pal.22.7
stud.pal.3.287
upz.1.pg604-8b

**Comment by jcowey** [comment]
@LaviniaFerretti , can you produce that list again, please, with two columns?

First column as above, second column with the value (ddb-hybrid) of the @n in `<ref n="`

If you can, please do so and paste in the results in a new comment.
